src/sql/workbench/services/notebook/browser/notebookServiceImpl.ts

```diff
@@ -455,7 +455,7 @@ export class NotebookService extends Disposable implements INotebookService {
 			if (!providerDescriptor.instance) {
 				// Await extension registration before awaiting provider registration
 				try {
-					await this._extensionService.whenInstalledExtensionsRegistered;
+					await this._extensionService.whenInstalledExtensionsRegistered();
```

Comment:
Just be careful here because we would be changing behavior by correcting the code.

Reply:
Yeah - @chlafreniere  @kevcunnane I'll assume that if you don't say anything you're fine with this. Should be fine here though - if anything I would expect this would get rid of possible race conditions.